### PR TITLE
Add new icons (“alert-diamond” + “alert-diamond-fill” + “closed-caption” + “info-fill”)

### LINKED ISSUE #37: Flight icon request: [closed-caption]
**Outline the use case for the new icon.**
We have a closed caption icon our video player. For DevDot we would like to replace the icons used on the existing player in order to be more consistent. We have everything we need except for a closed caption icon. Happy to use something else if you have other suggestions.

**If there's a similar existing icon, why isn't it a good fit? Please describe.**
Nothing really similar, we could house the CC options in the settings icon but I am not sure how much control we have. It would also require a bit of discovery work for users in order to find the CC controls.

**Describe alternatives you've considered.**
A button could work but would increase the height of the control bar.

**Screenshots**
image

**Figma**
[Add links to the relevant Figma files where this icon is used, if possible.](https://www.figma.com/file/VD7ahvXuXWJApeGnhbW4hv/Dev-Portal?node-id=6297%3A71283)

[Page with Learn's video player](https://learn-9qu12105v-hashicorp.vercel.app/tutorials/terraform/infrastructure-as-code?in=terraform/aws-get-started)

**Additional context**
N/A

### LINKED ISSUE #121: Flight icon request: [info-fill]
**Outline the use case for the new icon.**
A fill version of the **info** icon.

**If there's a similar existing icon, why isn't it a good fit? Please describe.**
Compact alerts need filled icons. The only one available for information is outlined. 

**Describe alternatives you've considered.**
None.

**Screenshots**
Screen Shot 2022-03-25 at 12 24 44 AM

**Figma**
[Alert component (WIP)](https://www.figma.com/file/noyY6dUMDYjmySpHcMjhkN/branch/Ru0h1oqy6r7SeGxwtQ6085/HDS-Product---Components-%5BWIP%5D?node-id=5241%3A18767)

**Additional context**
N/A

### LINKED ISSUE #122: Flight icon request: [alert-diamond] [alert-diamond-fill]
**Outline the use case for the new icon.**
We need an alert icon with a 4th shape. We have circle, triangle, and hexagon, we need another shape for color-blind users that can't rely on color.

**If there's a similar existing icon, why isn't it a good fit? Please describe.**
For compact alerts, alert-diamond and alert-diamond-fill icons are needed for accessibility purposes.

**Describe alternatives you've considered.**
alert-hexagon, but still similar to info.

**Screenshots**
Screen Shot 2022-03-25 at 1 22 40 PM

**Figma**
[Alert component (WIP)](https://www.figma.com/file/noyY6dUMDYjmySpHcMjhkN/branch/Ru0h1oqy6r7SeGxwtQ6085/HDS-Product---Components-%5BWIP%5D?node-id=1377%3A11987)

**Additional context**
N/A

### PR DESCRIPTION
**Pull request by didoo**
### :pushpin: Summary

This resolves #37
This resolves #121
This resolves #122

### :hammer_and_wrench: Detailed description

In this PR I have:
- synced local “flight-icons” assets with source Figma file
  - the following icons have been added `alert-diamond` + `alert-diamond-fill` + `closed-caption` + `info-fill` while other had some minor changes in their paths so appeared as modified (it's normal with Figma)
- regenerated the assets in output

Preview: https://hds-flight-website-git-new-icons-hashicorp.vercel.app/

### :camera_flash: Screenshots

screenshot_1202
screenshot_1201
screenshot_1200

***

### 👀 How to review

👉 Review by files changed

Reviewer's checklist:

- [ ] +1 Percy if applicable
- [ ] Confirm that PR has a changelog update via [Changesets](https://github.com/changesets/changesets) if needed

:speech_balloon: Please consider using [conventional comments](https://conventionalcomments.org/) when reviewing this PR.

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1202040332869498

### CODE REVIEW

File: .changeset/forty-goats-switch.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+"@hashicorp/flight-icons": minor
+---
+
+Added new icons “alert-diamond” + “alert-diamond-fill” + “closed-caption” + “info-fill”
```
Comment (amyrlam) [inline .changeset/forty-goats-switch.md:5]: q: When you wrote "regenerated the assets in output", given that it changes the paths of many other icons, should it be mentioned here? Realize it's not a user-facing change, but it's somewhat hard to follow what "regenerated the assets in output" means. 

Looking at this for `info-fill`, it looks a-ok to me otherwise

Comment (didoo) [reply .changeset/forty-goats-switch.md:5]: > When you wrote "regenerated the assets in output", given that it changes the paths of many other icons, should it be mentioned here?

It's an issue with Figma. They sometimes change internals of how they generate the SVG paths, and it happens 90% of the times there are some changes. So it would be in every release note about icons. These are no visible changes for the user. And there is no way to test these changes and see if they are breaking somehow:
- manually, is not possible (how do you compare two SVG internal paths?)
- automatically, it could be done with Percy, but testing 1000 icons every time (1000 screenshots) is VERY expensive! (the only solution would be to have a custom tool for visual regression testing; that's what I was using in my prev company, and we tested each asset independently so we could catch single unexpected icon changes).

**Review by cveigt — APPROVED** [review]
LGTM 👍
